Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/transformation/TransformationSaveService.java

```diff
@@ -0,0 +1,92 @@
+package edu.asu.spring.quadriga.service.impl.transformation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.workspace.ITransformationFile;
+import edu.asu.spring.quadriga.exceptions.FileStorageException;
+import edu.asu.spring.quadriga.service.transformation.ITransformationSaveService;
+import edu.asu.spring.quadriga.utilities.IFileSaveUtility;
+
+/**
+ * This service contains method to save contents of transformation files to
+ * local file storage
+ * 
+ * @author yoganandakishore
+ *
+ */
+
+@Service
+public class TransformationSaveService implements ITransformationSaveService {
+
+    @Qualifier("transformfileSaveUtil")
+    @Autowired
+    private IFileSaveUtility transformationFileManager;
+
+    @Override
+    public boolean saveFileToLocal(ITransformationFile transformationFile) throws FileStorageException {
+
+        String directoryName = getTransformationDirectoryName(transformationFile);
+        String metaDataContent = getMetaDataContent(transformationFile);
+
+        return (transformationFileManager.saveFiletoDir(directoryName, "metadata.txt", metaDataContent.getBytes())
```

Comment:
remove this long return statement. do the things you need to do, step by step and then return a result value. this is very hard to read.